src/pe_reports/report_generator.py

```diff
@@ -76,9 +87,9 @@ def convert_to(folder, source, timeout=None):
         folder,
         source,
     ]
-    process = subprocess.run(
-        args, stdout=subprocess.PIPE, stderr=subprocess.PIPE, timeout=timeout
-    )
+    process = subprocess.run(  # nosec
+        args, stdout=subprocess.PIPE, stderr=subprocess.PIPE, timeout=timeout  # nosec
+    )  # nosec
```

Comment:
I believe this is all that's strictly necessary.

```suggestion
    process = subprocess.run(
        args, stdout=subprocess.PIPE, stderr=subprocess.PIPE, timeout=timeout  # nosec
    )
```

Reply:
@mcdonnnj , @schmelz-ctr 

I updated my local code per this suggestion and flake8 fails. I believe that for the tests to pass the # nosec has to remain in the code.

Reply:
@mcdonnnj, @cduhn17 - I can confirm, between the flake8 reformatting of the long line and bandit's warning of `subprocess` it appears the interpretation `#nosec` needs to present on each line. 

Can I suggest we keep the original code to pass bandit errors for now and to keep in scope of PR, and create a TODO and new issue addressing bandit and subprocess in this instance.

Reply:
@schmelz-ctr Based on the arguments you are passing this call to `subprocess.run()` should have the `shell=False` option. Once that option is added the recommendation I made should hold true.

Reply:
looking - thanks

Reply:
That was it.. Resolved ae9a22f

Reply:
@mcdonnnj , 

Looking for your approval that your request has been satisfied. See commit [ae9a22f](https://github.com/cisagov/pe-reports/commit/ae9a22fea9a0044b62a0e173a658302110bd3e66). Thanks